avar2.md

```diff
@@ -16,39 +16,41 @@ Use cases include:
 * parametric fonts with intuitive control methods and a much reduced data footprint;
 * offering simpler methods of control for specialized variable fonts.
 
-# Table format
-
-Axis variation table:
+# avar version 2.0 header format
 
 | Type	| Name	| Description |
 | ------|-------|-------------|
 | `uint16`	| `majorVersion`	| Major version number of the axis variations table — set to 2. |
 | `uint16`	| `minorVersion`	| Minor version number of the axis variations table — set to 0. |
 | `uint16`	| `<reserved>`	| Permanently reserved; set to zero. |
-| `uint16`	| `axisCount`	| The number of variation axes for this font. This must be the same number as axisCount in the 'fvar' table. |
+| `uint16`	| `axisSegmentMapCount`	| The number of *axisSegmentMap*s for this font. If this is not 0, it must be the same as *axisCount* in the 'fvar' table. |
```

Comment:
Oops, this is in an intermediate state. Will fix. Definitely keep to *axisCount* for now?